### DISCUSSION TITLE
Order reports time zone

### DISCUSSION THREAD

**Discussion by Peymanpn** [amzn/selling-partner-api-models#3625] [General] [answered]
Hey there, Amazon team,

You guys are awesome, but I have a question. I'm trying to get some order reports from your site, but they don't match what I see on the UI. What time zone are these reports in? 

Thanks for your help.

**Comment by tspicer** [comment, marked as answer]
@Peymanpn A little late here, but Amazon uses UTC. There may be some exceptions, but see our docs for details:
[Understanding Timezones Within Amazon Seller Central and Amazon Advertising](https://docs.openbridge.com/en/articles/5053618-understanding-timezones-within-amazon-seller-central-and-amazon-advertising)
